### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by demkodo**
-----
[View rendered activities/office-management/index.md](https://github.com/publiccodenet/about/blob/ops-handbook-office-management/activities/office-management/index.md)

### CODE REVIEW

File: activities/office-management/index.md [hunk 1]
```diff
@@ -2,8 +2,15 @@
...
 
-* [Using our space guide](using-space.md): What roles spaces have and what their code of conduct is
+The office is located at:
+Keizersgracht 617
+1017 DS Amsterdam
+
+We have an anti-squat contract (Bruikleenovereenkomst) with: K&O Group who manages the property on behalf of the owner, Hansainvest GmbH. The contract ends on 31 March 2021 with the possibility of extending it for on a month-to-month basis, until the building has found a new renter.
+
+* [Using our space guide](using-space.md): What roles spaces have and what their code of conduct is.
+* Signed contract, office handbook (information on keys, WiFi, trash schedule) and contact details of K&O Group [are on the Secret Drive.](https://drive.google.com/drive/folders/1PJfus4U_eGuHkFPrxW1TTHH8RRg4tGpi)
```
Comment (ElenaFdR) [inline activities/office-management/index.md:16]:
```suggestion
* Signed contract, office handbook (information on keys, WiFi, trash schedule) and contact details of K&O Group [are on the Secret Drive](https://drive.google.com/drive/folders/1PJfus4U_eGuHkFPrxW1TTHH8RRg4tGpi).
```

File: activities/office-management/index.md [hunk 2]
```diff
@@ -2,8 +2,15 @@
...
+# Office and building management
 
 Our office houses the beating heart of the organization. We're all responsible for keeping it working and enjoyable.
 
-* [Using our space guide](using-space.md): What roles spaces have and what their code of conduct is
+The office is located at:
+Keizersgracht 617
+1017 DS Amsterdam
+
+We have an anti-squat contract (Bruikleenovereenkomst) with: K&O Group who manages the property on behalf of the owner, Hansainvest GmbH. The contract ends on 31 March 2021 with the possibility of extending it for on a month-to-month basis, until the building has found a new renter.
```
Comment (ElenaFdR) [inline activities/office-management/index.md:13]:
```suggestion
We have an anti-squat contract (bruikleenovereenkomst) with K&O Group, which manages the property on behalf of the owner, Hansainvest GmbH. The contract ends on 31 March 2021 with the possibility of extending it on a month-to-month basis, until the building has found a new renter.
```

**Review by ElenaFdR — COMMENTED** [review]
Small linguistic suggestions; no major issues. I *love* that we're making open the owner of our building - I've always wondered for large rented office spaces.

**Review by lscheske — APPROVED** [review]
* Committed Elena's editorial changes
* Good to go!

**Pull request merged**